db.json

```diff
@@ -428,7 +436,8 @@
     "compressible": true
   },
   "application/dicom": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["dcm"]
   },
```

Comment:
This is correct - I think the quotes in https://www.iana.org/assignments/media-types/application/dicom are causing this extension to be omitted from mime-db at the moment.